city_scrapers/spiders/chi_ssa_21.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+from city_scrapers.spider import Spider
+from city_scrapers.constants import COMMISSION
+import dateutil.parser
+
+
+class ChiSsa21Spider(Spider):
+
+    name = 'chi_ssa_21'
+    agency_name = 'Lincoln Square Neighborhood Improvement Program'
```

Comment:
The agency would be "Chicago Special Service Area #21 Lincoln Square Ravenswood". SSAs get complicated when they have the nonprofit administrator, so we're just going with the SSA name here